src/server/storage-controller.js

```diff
@@ -55,6 +57,12 @@ class StorageController {
       throw new BadRequest(`Invalid storage path '${prefix}'`);
     }
 
+    const requiredBytes = files.reduce((totalSize, file) => totalSize + file.size, 0);
```

Comment:
I had actually something different in mind: here the bytes are already saved on the server (and could potentially put it under pressure. We should set a limit in the `multipartParser` middleware to not even accept more than the number of bytes. Sorry for having been too vague about it in my last comment.